package.json

```diff
@@ -116,6 +116,9 @@
     "travis-deploy-once": "^5.0.9",
     "ts-node": "^8.2.0"
   },
+  "resolutions": {
+    "acorn": "6.3.0"
```

Comment:
Out of curiosity @markdalgleish - isn't the Babel parser like 100x better?

Reply:
This is for Playroom, primarily, which runs in the browser. We still use Babel.